test cases/common/131 override options/meson.build

```diff
@@ -1,6 +1,12 @@
 project('option override', 'c',
-  default_options : 'unity=on')
+  default_options : [
+    'unity=on',
+    'unity_size=2'])
 
-executable('mustunity', 'one.c', 'two.c')
-executable('notunity', 'three.c', 'four.c',
+slib_notinstalled = static_library('slib_notinstalled', ['slib1.c', 'slib2.c', 'slib3.c', 'slib4.c'])
```

Comment:
I think you should move this test into its own test case, here we test that `unity=off` override works which is a completely different thing.

We already have a CI job that runs all tests from common/ with unity=on, so a new test could simply be skipped when unity is not enabled:
```meson
if get_option('unity') != 'on'
  error('MESON_SKIP_TEST testing only unity build')
endif
```

If I understand correctly the issue, only 2 files is enough.

The fix itself makes sense, nice little math error, good catch.